product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx

```diff
@@ -284,6 +289,104 @@ BEGIN
     UTL_FILE.FCLOSE(v_empfile);
 END;
 ```
+## FGETATTR
+
+The `FGETATTR` function retrieves the attributes for a given file. When you enter a `filename` and `location`, it returns whether the file exists. If the file exists, it also returns the `file_length` and `block_size`. If the file doesn't exist or there is an error, the attributes return as `NULL`. 
```

Comment:
```suggestion
The `FGETATTR` function retrieves the attributes for a given file. When you provide a `filename` and `location`, it returns whether the file exists. If the file exists, it also returns the `file_length` and `block_size`. If the file doesn't exist or there is an error, the attributes return as `NULL`. 
```

Reply:
The `FGETATTR` function retrieves the attributes for a given file. When you provide `filename` and `location` values, it returns whether the file exists. If the file exists, it also returns `file_length` and `block_size` vakyes. If the file doesn't exist or an error occurs, the attributes return as `NULL`.